game-core/src/main/java/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -48,15 +48,19 @@
   public static final String PROPERTY_UNITS_SCALE = "units.scale";
   public static final String PROPERTY_UNITS_WIDTH = "units.width";
   public static final String PROPERTY_UNITS_HEIGHT = "units.height";
+  public static final String PROPERTY_UNITS_COUNTER_OFFSET_WIDTH = "units.counter.offset.width";
+  public static final String PROPERTY_UNITS_COUNTER_OFFSET_HEIGHT = "units.counter.offset.height";
+  public static final String PROPERTY_UNITS_STACK_SIZE = "units.stack.size";
+  public static final String PROPERTY_UNITS_COLOR_PREFIX = "units.color.";
+  public static final String PROPERTY_UNITS_COLOR_BRIGHTNESS_PREFIX = "units.color.brightness.";
+  public static final String PROPERTY_UNITS_COLOR_FLIP_PREFIX = "units.color.flip.";
```

Comment:
Both color and flip uses the same ignore list as I had a hard time imagining you wanting 2 separate ignore lists. Generally, I don't users will be flipping images unless you are already coloring them. I consider it more of an extension of coloring images than a separate feature.

Reply:
Ah, didn't realize they were using the same ignore list. I personally did not draw the same connection that the two were related. "color.flip" I would interpret naively as inverting the color, not changing the orientation.

Is it possible to use 'flip' without using colors? It seems the two concepts should maybe not be so coupled. It might be okay maybe to duplicate some ignore logic to have two independent lists in case we do want to start coloring factories but not flip them. I wonder if that level of flexibility would actually be a good thing.

Reply:
Open to naming it something besides flip. I don't think there are many actual use cases for flipping units where you aren't colorizing them so separate lists I think are unnecessary and starting to over engineer it.

Reply:
Personally I'm like for the prefix to be `units.image.*`

Perhaps one of these may strike you, some suggestions I could think of:
```
units.flip.horizontal=true
units.rotate.horizontal=true
units.image.rotate=180
units.image.rotate.horizontal=true
units.image.rotate=horizontal
```

Reply:
So rotate is actually a different image operation: https://www.fotojet.com/features/photo-editor/rotate-image.html. I'll think about remaining these properties a bit. I think the main issue is actually just the prefix and the reason is I added flip at the very end. I think it should probably be something like `units.transform` or `units.image`. Essentially the same change I made where I originally have the class `ImageColorizer` and then changed it to `ImageTransformer`.